src/SDPoptimize.jl

```diff
@@ -237,6 +237,8 @@ function sdp_solve_DH(model::StochDynProgModel,
     #Compute cartesian product spaces
     product_states, product_controls = generate_grid(model, param)
 
+    product_controls = collect(product_controls)
```

Comment:
Me neither ^^.
I have to test if it does the same thing with product_states on instances with large time steps set (on small examples there is no difference).